tests/ext/crashtracker_segfault.phpt

```diff
@@ -0,0 +1,71 @@
+--TEST--
+Send crashtracker report when segmentation fault signal is raised and config enables it
+--SKIPIF--
+<?php
+if (!extension_loaded('posix')) die('skip: posix extension required');
+if (getenv('SKIP_ASAN') || getenv('USE_ZEND_ALLOC') === '0') die("skip: intentionally causes segfaults");
+if (getenv('PHP_PEAR_RUNTESTS') === '1') die("skip: pecl run-tests does not support XFAIL");
```

Comment:
XFAIL? Is it because of `%A`?

Reply:
yes, it's because of `%A`, I updated the comment